docs/logql.md

```diff
@@ -1,26 +1,28 @@
 # LogQL: Log Query Language
 
 Loki comes with its own PromQL-inspired language for queries called *LogQL*.
-LogQL can be considered a distributed `grep` that aggregates log sources and
+LogQL can be considered a distributed `grep` that aggregates log sources,
 using labels and operators for filtering.
 
-There are two types of LogQL queries: *log queries* which return the contents of
-log lines, and *metric queries* which extend log queries and calculates values
-based on the counts of logs from a log query.
+There are two types of LogQL queries: 
 
-A basic log query consists of two parts: the **log stream selector** and a
-**filter expression**. Due to Loki's design, all LogQL queries are required to
-contain a log stream selector.
+-*log queries* return the contents of log lines.
+-*metric queries* extend log queries and calculate values based on the counts of logs from a log query.
```

Comment:
```suggestion
-*Metric queries* extend log queries and calculate values based on the counts of logs from a log query.
```

Reply:
Yep. Thanks for pointing out!

Reply:
@oddlittlebird I checked out the master doc to understand my choices and saw that the bolds and itallics that I was using are the same that were in already there. Capitalizing "-Metric queries looks good, but then we'd have to do the same for "-Log queries" that appears right before it. What do you think?

Reply:
These look good - going to commit them to get this all merged. Thanks for your first contribution 🥇